### ISSUE TITLE
rusty-tags fails silently

### ISSUE THREAD

**Issue by covercash2** [dan-t/rusty-tags#46]
i'm not sure why it's failing or exactly how to reproduce the bug. i'm using the default config, trying to generate tags for remacs source.

```
$ cd code/remacs
$ rusty-tags -v -s ./rust_src emacs    # also fails with vi
Using configuration: vi_tags='rusty-tags.vi', emacs_tags='rusty-tags.emacs', ctags_options=''
Found Excuberant Ctags with executable 'ctags'
Fetching source and metadata ...
```
i've been able to generate tags for other projects.

**Comment by dan-t** [comment]
Should now work with `rusty-tags 2.8.0`.

**Comment by dan-t** [comment]
There has still been an issue with the `remacs-lib` crate which should now be fixed with `rusty-tags 2.9.0`.
